src/frontend/src/views/Data/index.tsx

```diff
@@ -37,26 +38,25 @@ const Data: FunctionComponent = () => {
 
   const router = useRouter();
 
+  const treeResponse = useTreeInfo();
+  const { data, isLoading } = treeResponse;
```

Comment:
it's a little weird to me that this is outside of the useEffect hook below, but that fetchSamples is not, do you think we could remove the fetchSamples from the useEffect hook as well? Or is there a reason why that still has to be there?

Reply:
can we take useTreeInfo out because of the react query queryKey parameter, so it only changes when the key hash is updated?